### ISSUE TITLE
what is the relationship between validation and training ?

### ISSUE THREAD

**Issue by wangchuanxd** [vlfeat/matconvnet#135]
Hi. Why do we add validation while training a model ? Is there direct quantity relationship between validation and training ? Does someone know that ? Thanks a lot !

**Comment by johnny5550822** [comment]
Validation set is a separate data set that IS NOT used during training. but then will be tested (classified) while your classifier is being trained in order to make sure that your classifier is working properly (e.g., not overfitting)

There is a test set, which once your model build, you will test your model on it. Validation set is used during training; test set is used during testing. 

Hope this helps.

**Comment by wangchuanxd** [comment]
@johnny5550822  Oh, I see. Thanks!
